modules/tinymce/src/core/test/ts/browser/content/EditorContentNotInitializedTest.ts

```diff
@@ -34,14 +29,14 @@ describe('browser.tinymce.core.content.EditorContentNotInitializedTest', () => {
 
   it('set content on editor without initializing it', () => {
     const editor = createEditor();
-    setContentAndAssertReturn(editor, 'hello');
+    editor.setContent('hello');
```

Comment:
This isn't testing the same thing anymore. The test would check if setContent on an editor would return the same value on getContent if the editor was removed or not yet initialized.

- [ ] Revert test change since this is no longer asserting anything

Reply:
Yes. The deprecated thing here was that the returned value was removed. Checking the return value after that would always be `void`.

Reply:
Right I see that the type is changed and the return. Where did we list that as something we want to break?

Reply:
editor.ts, line 809 `<em>Note: The content return value was deprecated in TinyMCE 6.0 and has been marked for removal in TinyMCE 7.0.</em>`

Reply:
Right yes that and the setContent API change needs to be reverted. We probably should update that incorrect comment to mention 8.0 since the 7.0 ship has sailed.